ansible/runload.yaml

```diff
@@ -4,9 +4,7 @@
   hosts: loadrunners
   vars:
     ansible_host_key_checking: false
-    validator_0_ip: "{{ groups['validators'][0] }}"
-    validator_0_internal_ip: "{{ hostvars[validator_0_ip].internal_ip }}"
-    endpoints: ws://{{ validator_0_internal_ip }}:26657/v1/websocket # comma separated list of endpoints
+    endpoints: ws://{{ hostvars[groups[validators][0]].internal_ip }}:26657/v1/websocket # comma separated list of endpoints
```

Comment:
I don't actually mind assigning some variables for readability, just the way it was done looked a bit inconsistent. It was a small nitpick, anyway.